plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -279,13 +279,25 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get").Groups(legacyGroup).Resources("endpoints", "namespaces", "serviceaccounts").RuleOrDie(),
 				rbac.NewRule("update").Groups(legacyGroup).Resources("endpoints", "serviceaccounts").RuleOrDie(),
 
-				rbac.NewRule("list", "watch").Groups("*").Resources("namespaces", "nodes", "persistentvolumeclaims",
-					"persistentvolumes", "pods", "secrets", "services", "serviceaccounts", "replicationcontrollers").RuleOrDie(),
+				rbac.NewRule("list", "watch").Groups(legacyGroup).Resources(
```

Comment:
Because the kube controller manager's identity is the client that is used to run the shared informers, and it needs to be able to list/watch everything used by all the controllers.